templates/elixir-ci.yaml

```diff
@@ -14,6 +14,8 @@ env:
   ELIXIR_VERSION: 1.13.4
   OTP_VERSION: 24
   MIX_ENV: test
+  ELIXIR_VERSION_OBS: 1.13.4
```

Comment:
we could remove the env here and use the matrix directly

I would say we could  use the latest elixir version with the latest OTP, and the obs elixir version + the obs otp version.

Reply:
I didn't do that because in the `web` repo's [ci.yaml](https://github.com/trento-project/web/blob/main/.github/workflows/ci.yaml) for example, there is a `test` step and `test-e2e` step for example, and we would have to maintain the versions across both steps.

Reply:
(assuming that we are running both versions on both steps, is that correct @fabriziosestito?)